scripts/audit/find_real_orphans.py

```
@@ -0,0 +1,384 @@
+def is_git_tracked(path: Path, repo_root: Path) -> bool:
+    """检查文件是否被 git 追踪 (排除 untracked 工作区幽灵).
+
+    安全降级: git 命令不可用 / 非仓库等异常时返回 True (保守保留),
+    避免因环境问题误判为孤儿。
+    """
+    try:
+        rel = path.relative_to(repo_root)
+    except ValueError:
+        return True  # 路径异常时保守保留
+    try:
+        result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)